### PR TITLE
Dev 876 add moonbeam to the known networks

### PR DESCRIPTION
**Pull request by MarcusviniciusLsantos**

### CODE REVIEW

File: helpers/networks.ts [hunk 1]
```diff
@@ -101,6 +101,24 @@ export const NETWORKS: {
...
+    name: "Moonbase Testnet",   
+    key: "moonbase",
+    currency: ETH,
+    decimals: 18,
+    explorerURL: "https://blockscout.moonriver.moonbeam.network",
+    rpcUrls: ["https://rpc.api.moonbase.moonbeam.network"]
+  },
+  "0x504": {
+    id: "1284",
+    name: "Moonbeam Testnet",
```
Comment (hvasconcelos) [inline helpers/networks.ts:115]: Moonbeam is a main net

File: helpers/networks.ts [hunk 2]
```diff
@@ -101,6 +101,24 @@ export const NETWORKS: {
...
+    rpcUrls: ["https://rpc.api.moonbase.moonbeam.network"]
+  },
+  "0x504": {
+    id: "1284",
+    name: "Moonbeam Testnet",   
+    key: "moonbeam",
+    currency: ETH,
+    decimals: 18,
+    explorerURL: "https://blockscout.moonriver.moonbeam.network",
+    rpcUrls: ["https://rpc.api.moonbase.moonbeam.network"]
```
Comment (hvasconcelos) [inline helpers/networks.ts:120]: the RPC is wrong

File: helpers/networks.ts [hunk 3]
```diff
@@ -101,6 +101,24 @@ export const NETWORKS: {
...
+  },
+  "0x504": {
+    id: "1284",
+    name: "Moonbeam Testnet",   
+    key: "moonbeam",
+    currency: ETH,
```
Comment (hvasconcelos) [inline helpers/networks.ts:117]: currency is GLMR

File: helpers/networks.ts [hunk 4]
```diff
@@ -101,6 +101,24 @@ export const NETWORKS: {
...
+    decimals: 18,
+    explorerURL: "https://blockscout.moonriver.moonbeam.network",
```
Comment (hvasconcelos) [inline helpers/networks.ts:119]: explorer is also wrong

File: helpers/networks.ts (hunk 2 above)
Comment (MarcusviniciusLsantos) [reply helpers/networks.ts:126]: what is the correct?

image

File: interfaces/enums/network-colors.ts [hunk 5]
```diff
@@ -11,6 +11,8 @@ export enum NetworkColors {
   afrodite = "#29b6af",
   irene = "#29b6af",
   iris = "#29b6af",
+  moonbase = "#29b6af",
+  moonbeam = "#29b6af",
```
Comment (moshmage) [inline interfaces/enums/network-colors.ts:15]: moonbeam color should not be the same as dev networks, it should match `f6c343` (moonriver)

**Review by moshmage — CHANGES_REQUESTED** [review]

**Review by moshmage — APPROVED** [review]

**Pull request merged**